CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ The old behavior of the `full_pipeline` can be obtained by running `full_pipelin
 
 * The `crispr` and `hashing` modalities have been renamed to `gdo` and `hto` respectively ([#392](https://github.com/openpipelines-bio/openpipeline/pull/392)).
 
+* Updated viash to 0.7.4
```

Comment:
:raised_eyebrow:

Reply:
This change was introduced in to add this to https://github.com/openpipelines-bio/openpipeline/commit/c980573910451dff4667fde00fe06eab9e844d5c

But there was no CHANGELOG entry for it, so I am adding it in this PR